docs/pages/workflow/expo-cli.md

```diff
@@ -5,1083 +5,284 @@ maxHeadingDepth: 4
-| `-m, --host [mode]`                     | lan (default), tunnel, localhost. Type of host to use. "tunnel" allows you to view your link on other networks |
-| `--tunnel`                              | Same as --host tunnel                                                                                          |
-| `--lan`                                 | Same as --host lan                                                                                             |
-| `--localhost`                           | Same as --host localhost                                                                                       |
-| `--offline`                             | Allows this command to run while offline                                                                       |
-| `--dev`                                 | Deprecated: Dev mode is used by default                                                                        |
-| `--no-minify`                           | Deprecated: Minify is disabled by default                                                                      |
-| `--no-https`                            | Deprecated: https is disabled by default                                                                       |
-| `--config [file]`                       | Deprecated: Use app.config.js to switch config files instead.                                                  |
-
-</p>
-</details>
+- <kbd>a</kbd> Open the project in Expo Go on Android.
+- <kbd>shift+a</kbd> Select an Android device or emulator to open.
 
----
+- <kbd>i</kbd> Open the project in Expo Go on iOS.
+- <kbd>shift+i</kbd> Select an iOS Simulator to open.
+- <kbd>w</kbd> Open the project in a web browser. This may require Webpack to be installed in your project.
 
-### Auth
+- <kbd>r</kbd> Reload the app on any connected device.
+- <kbd>m</kbd> Open the dev menu on any connected native device (web not support).
+- <kbd>shift+m</kbd> Choose more commands to trigger on connected devices. This includes toggling the performance monitor, opening the element inspector, reloading the device, and opening the dev menu.
+- <kbd>j</kbd> Open Chrome Dev Tools for any connected device that is using Hermes as the JavaScript engine. [Learn more](/guides/using-hermes/).
+- <kbd>o</kbd> Open project code in your editor. This can be configured with the `EXPO_EDITOR` and `EDITOR` environment variables.
+- <kbd>c</kbd> Show development server URL as a QR code in the terminal.
+- <kbd>?</kbd> Show all Terminal UI commands.
 
-<details>
-<summary>
-<h4>expo login</h4>
-<p>Login to an Expo account</p>
-</summary>
-<p>
+### Server URL
 
-Alias: `expo signin`
+By default, the URL is served over a lan connection utilizing the local network. You can change this behavior to localhost-only by using the flag `--host localhost` or `--localhost`.
```

Comment:
```suggestion
The URL is served over a LAN connection that utilizes the local network by default. You can change this behavior to localhost-only by using the flag `--host localhost` or `--localhost`.
```

I think for abbreviations like this, it's better to stick with the most commonly used one. I'm not sure if this a common way to use in the docs.